src/cli.rs

```diff
@@ -34,352 +32,351 @@ lazy_static! {
         SIGSTORE_TARGETS_PATH.join("rekor.pub");
 }
 
-pub fn build_cli() -> clap::App<'static, 'static> {
+pub fn build_cli() -> clap::App<'static> {
     App::new(crate_name!())
         .version(crate_version!())
         .author(crate_authors!())
         .about(crate_description!())
-        .arg(Arg::with_name("verbose").short("v").help("Increase verbosity"))
+        .arg(Arg::new("verbose").short('v').help("Increase verbosity"))
         .subcommand(
-            SubCommand::with_name("policies")
```

Comment:
I was a bit suprised by this API changed on clap's side (SubCommand -> Into<App>). Seems that everything is working fine. I tried some commands, also with the generated `--help` and such and everything seems to work fine. e2e tests are passing as well.